src/table/Source.cc

```diff
@@ -175,12 +175,13 @@ void SourceFitsWriter::_writeRecord(BaseRecord const & r) {
 //----- SourceFitsReader ------------------------------------------------------------------------------------
 //-----------------------------------------------------------------------------------------------------------
 
-// A custom FitsReader for Sources - this reads footprints as variable-length arrays, and adds header
-// keys that define the slots.  It gets registered with name SOURCE, so it should get used whenever
-// we read a table with AFW_TYPE set to that value.
+// A custom FitsReader for Sources - this reads footprints in addition to the regular fields.  It
```

Comment:
There are actually a couple of moderately weighty classes defined between here and ``SourceFitsReader``. They're necessary to make things work, but maybe this comment could be a bit closer to the class it actually describes?